openmsupply/open-msupply
# 1701 Pending approval response requisition appears as editable in list

Fixes #1701

# 👩🏻‍💻 What does this PR do? 
Grey out requisitions that are pending or denied in Requisition View

# 🧪 How has/should this change been tested? 
- Have a pending or denied requisition
- Click on Requisition in sub-menu
- See that requisitions which are pending or denied is greyed out
